app/src/main/java/com/ncnf/models/Group.java

```diff
@@ -16,15 +16,29 @@
     public FirebaseDatabase firebaseDatabase;
 
     private final List<String> invited;
+    private final  List<String> attendees;
```

Comment:
We had a discussion in a meeting about the attributes invited and attendees. I think we concluded that a group should have only one attribute (e.g. members).

Reply:
I think this will be change in another PR so it's normal that it is here now.